tests/js/spec/components/contextPickerModal.spec.jsx

```diff
@@ -248,4 +249,179 @@ describe('ContextPickerModal', function () {
     await tick();
     wrapper.unmount();
   });
+
+  it('isSuperUser and selects an integrationConfig and calls `onFinish` with URL to that configuration', async function () {
+    OrganizationsStore.load([org]);
+    OrganizationStore.onUpdate(org);
+    ConfigStore.config = {
+      user: {isSuperuser: true},
+    };
+
+    const provider = {slug: 'github'};
+    const configUrl = `/api/0/organizations/${org.slug}/integrations/?provider_key=${provider.slug}&includeConfig=0`;
+    const githubConfigs = [
+      {
+        accountType: 'User',
+        configData: null,
+        domainName: 'github.com/NisanthanNanthakumar',
+        icon: 'https://avatars.githubusercontent.com/u/10494',
+        id: '1',
+        name: 'NisanthanNanthakumar',
+        provider: {
+          aspects: {},
+          canAdd: true,
+          canDisable: false,
+          features: ['codeowners', 'commits', 'issue-basic', 'stacktrace-link'],
+          key: 'github',
+          name: 'GitHub',
+          slug: 'github',
+        },
+        status: 'active',
+      },
```

Comment:
Can we use/update https://github.com/getsentry/sentry/blob/master/tests/js/sentry-test/fixtures/githubIntegration.js?